lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -224,65 +233,61 @@ export default class BitcoinProcessor {
 
   private async processBlockData (
     blockData: {[blockHash: string]: BitcoinBlockModel},
-    chainInfos: Map<string, IBlockInfoExtended>[],
+    notYetValidatedBlocks: Map<string, IBlockInfoExtended>,
     startingBlockHeight: number,
     heightOfEarliestKnownValidBlock: number) {
 
     for (let blockHash in blockData) {
       const block = blockData[blockHash];
 
       if (block.height >= startingBlockHeight && block.height <= heightOfEarliestKnownValidBlock) {
-        const indexInChainInfos = block.height - startingBlockHeight;
-        BitcoinProcessor.addBlockToChainInfo(chainInfos, indexInChainInfos, block);
+        const blockReward = BitcoinProcessor.getBitcoinBlockReward(block.height);
+        notYetValidatedBlocks.set(
+          block.hash,
+          {
+            height: block.height,
+            hash: block.hash,
+            totalFee: block.transactions[0].outputs[0].satoshis - blockReward,
+            transactionCount: block.transactions.length - 1, // minus one because of coinbase
```

Comment:
Discussion: I'd rather `transactionCount` to be the same as `transactions.length` so no context knowledge required, of course it would skew the fee by 1/3000th, but not big issue I think.

Reply:
Fair